globals.json

```diff
@@ -745,5 +748,16 @@
 		"Path": false,
 		"Progress": false,
 		"Ref": false
+	},
+	"serviceworker": {
+		"Cache": false,
+		"CacheStorage": false,
+		"Client": false,
+		"Clients": false,
+		"ExtendableEvent": false,
```

Comment:
Is this really a global? If so, then I think `ExtendableMessageEvent` is missing.